sunpy/map/mapbase.py

```diff
@@ -1271,10 +1271,19 @@ def scale(self):
         """
         Image scale along the x and y axes in units/pixel
         (i.e. cdelt1, cdelt2).
+
+        If the CDij matrix is defined but no CDELTi values are explicitly defined,
+        effective CDELTi values are constructed from the CDij matrix.
```

Comment:
I think it would be good to explain *how* here, because as you pointed out in #6570 there's not one singular "correct" way to do this. (Which is why it didn't used to do it)

Reply:
Do you want this in this PR or another one?

Reply:
This one preferably.

Reply:
+1 to documenting this clearly in this PR.

Reply:
Elaborated